versioned_docs/version-5.x/hiding-tabbar-in-screens.md

```diff
@@ -38,7 +38,7 @@ function HomeTabs() {
     <Tab.Navigator>
       <Tab.Screen name="Home" component={Home} />
       <Tab.Screen name="Feed" component={Feed} />
-      <Tab.Screen name="Settings" component={Settings} />
+      <Tab.Screen name="Notifications" component={Notifications} />
```

Comment:
```suggestion
      <Tab.Screen name="Settings" component={Settings} />
      <Tab.Screen name="Notifications" component={Notifications} />
```

Settings screen is still mentioned in the doc 😃

Reply:
Settings screen is still in App component, so I don't think it should be duplicated here.